internal/pkg/skuba/addons/gangway_test.go

```diff
@@ -91,11 +91,9 @@ func Test_renderContext_GangwayImage(t *testing.T) {
 func Test_renderContext_GangwayClientSecret(t *testing.T) {
 	for _, ver := range kubernetes.AvailableVersions() {
 		t.Run("get gangway client secret when cluster version is "+ver.String(), func(t *testing.T) {
-			fake.NewSimpleClientset()
-
 			got := renderContext{}.GangwayClientSecret()
-			if got != "" {
-				t.Errorf("expect got client secret empty")
+			if got == "" {
+				t.Errorf("expect got non-empty client secret")
```

Comment:
NIT: (don't need to fix but if there is something else you already are changing the code for)
`got` in this context is confusing `expected non-empty client secret` the original was a rough read too. :) so no worries! I think it originally was meaning to substitute in the var `got` but... it's also in a test so really don't worry about it if you don't need to update this PR.